app/src/main/java/app/revanced/integrations/patches/components/PlayerFlyoutMenuItemsFilter.java

```diff
@@ -0,0 +1,72 @@
+package app.revanced.integrations.patches.components;
+
+import android.os.Build;
+
+import androidx.annotation.RequiresApi;
+
+import app.revanced.integrations.settings.SettingsEnum;
+import app.revanced.integrations.utils.ReVancedUtils;
+
+public class PlayerFlyoutMenuItemsFilter extends Filter {
+    private static String[] exceptions;
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    public PlayerFlyoutMenuItemsFilter() {
+        exceptions = new String[]{
+                "comment", // Anything related to comment section.
+                "CellType|", // Prevent comment filter from being hidden on top of comment section.
+                "_sheet_", // Comment flyout panel for reporting and replying options.
+                "video_with_context" // Prevent video player lags sometimes when new video is started.
```

Comment:
What do you mean by video player lags? Why is the video player lagging

Reply:
When video player is started, filtering begins. That sometimes causes it to lag.

Reply:
This may be important for all protobuf buffer filters